modules/whosOnFirst.py

```diff
@@ -77,20 +79,22 @@ def get_svg(self, led):
             svg += f'<text x="{x}" y="{y}" text-anchor="middle" style="font-family:sans-serif;font-size:16pt;" fill="#000">{text}</text>'
         svg += '</svg>'
         return svg
-    
+
     def randomize(self):
         self.display = random.choice(list(self.DISPLAY_WORDS.keys()))
         self.buttons = random.sample(random.choice(self.BUTTON_GROUPS), 6)
         self.log(f"State randomized. Stage {self.stage}. Display: {self.display}. Buttons: {' '.join(self.buttons)}")
-    
+
+    def button_replace_values(self, v):
+        return v
+
     @modules.check_solve_cmd
     async def cmd_push(self, author, parts):
         if not parts:
             return await self.usage(author)
         button = ' '.join(parts).upper()
 
-        if self.third_base:
-            button = button.replace('0', 'O').replace('1', 'I')
+        button = self.button_replace_values(button)
```

Comment:
Consider merging this into one line: `button = self.preprocess_input(' '.join(parts).upper())`

Reply:
also for an outdated commit